app/views/moderations/_mod_sidebar_left.html.erb

```diff
@@ -27,14 +27,15 @@
 
     <div id="mod-center-inbox" class="mod-center--nav--default crayons-tabs crayons-tabs--scrollable flex m:hidden">
```

Comment:
(blocking) - I think the CSS changes may have affected this section (on the inactive tab):

![screenshot showing these tabs with a strange grey block under the inactive tab](https://user-images.githubusercontent.com/20773163/117009948-32d0d580-ace4-11eb-9fb4-1ed0c3355837.png)

Reply:
Ah, if there are multiple things in the tab header they have to be in span, fixed